app/controllers/api/account.php

```diff
@@ -494,6 +498,61 @@
                         'memberships' => null,
                         'search' => implode(' ', [$userId, $email, $name])
                     ])));
+
+                    /** Trigger Webhook */
+                    $userModel = new User();
+
+                    $userCreate = new Event(Event::WEBHOOK_QUEUE_NAME, Event::WEBHOOK_CLASS_NAME);
+                    $userCreate
+                        ->setProject($project)
+                        ->setUser($user)
+                        ->setEvent('users.[userId].create')
+                        ->setParam('userId', $userId)
+                        ->setPayload($user->getArrayCopy(array_keys(($userModel->getRules()))))
+                        ->trigger();
+
+                    /** Trigger Functions */
+                    $userCreate
+                        ->setClass(Event::FUNCTIONS_CLASS_NAME)
+                        ->setQueue(Event::FUNCTIONS_QUEUE_NAME)
+                        ->trigger();
+
+                    /** Trigger realtime event */
+                    $allEvents = Event::generateEvents('users.[userId].create', [
+                        'userId' => $userId
+                    ]);
+
+                    $target = Realtime::fromPayload(
+                        event: $allEvents[0],
+                        payload: $user
+                    );
+
+                    Realtime::send(
+                        projectId: 'console',
+                        payload: $user->getArrayCopy(),
+                        events: $allEvents,
+                        channels: $target['channels'],
+                        roles: $target['roles']
+                    );
```

Comment:
we don't need this event for the console 👍🏻

Reply:
Resolved. 👍